CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 lib_unity Change Log
 ====================
 
+2.6.0
+-----
+
+  * ADDED: cmake for automatically generating test runners for xcommon
```

Comment:
xcommon is the old build system; this should be XCommon CMake applications.

Reply:
```suggestion
  * ADDED: cmake for automatically generating test runners for XCommon CMake
```